servers/zms/src/main/java/com/yahoo/athenz/zms/purge/PurgeMember.java

```diff
@@ -0,0 +1,57 @@
+package com.yahoo.athenz.zms.purge;
+
+public class PurgeMember {
+    private String domainName;
+    private int collectionId;
```

Comment:
we should never expose the collectionId and principalId. The server does not know what the underlying data storage is thus it can't make assumptions about it. This is why the DB layer has its own internal cache